docs/contributing/api-reference.md

```
@@ -1,14 +1,254 @@
+
+# 2. Use the token in subsequent requests
+curl -H "Authorization: Bearer eyJ..." \
+  http://localhost:3003/api/config/yaml
+```
+
+---
+
+## Admin UI Backend Endpoints (Port 3003)
+
+### Auth (`/api/auth`)
+| Method | Endpoint | Description |
```

markdownlint-cli2 (0.20.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)